build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val plugins : Seq[Plugins] = Seq.empty
 lazy val playSettings : Seq[Setting[?]] = Seq.empty
 
 ThisBuild / majorVersion := 0
-ThisBuild / scalaVersion := "3.3.4"
+ThisBuild / scalaVersion := "3.6.4"
```

Comment:
Where is this version taken from? According to the Catalogue, the latest version is `3.3.5`.